doc/index.rst

```diff
@@ -2,6 +2,225 @@
  Welcome to Ceph
 =================
 
+Ceph is a storage platform. 
+
+Ceph makes possible object storage, block storage, and file storage. It can be used to build cloud infrastructure and web-scale object storage. 
+
+The procedure on this page explains how to set up a three-node Ceph cluster, the most basic of setups.  
+
+Basic Three-Node Installation Procedure
+=======================================
+.. highlight:: console 
+
+
+Installing the First Node
+-------------------------
+
+#. Install a recent, supported Linux distribution on a computer.
+#. Install docker. On Fedora or Centos::
+
+   $ sudo dnf install docker 
+
+  on Ubuntu or Debian::
+
+   $ sudo apt install docker.io
+
+#. Fetch the cephadm utility from github to the computer that will be the Ceph manager::
+
+   $ curl --silent --remote-name --location https://github.com/ceph/ceph/raw/master/src/cephadm/cephadm
```

Comment:
I would prefer to install cephadm using the RPM package, as this automatically resolves the dependencies, lilke podman, lvm2 etc.

Reply:
the goal was to simplify the procedure here - not requiring any extra repo setup or distro-specific-ness